pkg/common/cns-lib/volume/listview.go

```diff
@@ -241,6 +240,23 @@ func (l *ListViewImpl) RemoveTask(ctx context.Context, taskMoRef types.ManagedOb
 	return nil
 }
 
+func (l *ListViewImpl) SetListViewNotReady(ctx context.Context) {
+	log := logger.GetLogger(ctx)
+	l.mu.Lock()
```

Comment:
all placed where we are waiting got lock to get acquired, can you print log message and after lock is obtained print log message. make sure log message has method name.

Reply:
done.